### PR TITLE
OTP generator 

### PR DESCRIPTION
**Pull request by Daniel-Ojo-Williams**
OTP generator function and controller for otp verification

### CODE REVIEW

File: src/utils/apiStatusCode.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+const apiHttpStatusCodes = {
+  STATUS_OK: 200,
+  STATUS_CREATED: 201,
+  STATUS_UNAUTHORIZED: 401,
+  STATUS_FORBIDDEN: 403,
+  STATUS_NOT_FOUND: 404,
+  STATUS_CONFLICT: 409,
+  STATUS_BAD_REQUEST: 400,
```
Comment (Jhay-Great) [inline src/utils/apiStatusCode.js]: It's really nice how it's been abstracted, I usually hard code it all the time. So this is very subtle but for design purposes, it'll be nice if they are all in ascending order, it makes reading easier. Initially thought it was left out till I got here.

**Review by Jhay-Great — COMMENTED** [review]
Nice one boss, really liked otpGenerator.js. I haven't worked with it before but got the understanding easily. All the other files were great as well. Just made a suggestion I will appreciate you if you kindly check it out

**Review by Danbaba1 — COMMENTED** [review]
Changed single quotes to double quotes

File: src/utils/apiStatusCode.js (hunk 1 above)
Comment (Daniel-Ojo-Williams) [reply src/utils/apiStatusCode.js]: That's true, i'll make that change now. Thank you.

**Pull request merged**